lm-agent-snap/snap/snapcraft.yaml

```diff
@@ -0,0 +1,80 @@
+name: license-manager-agent
+base: core24
+version: '0.1.0'
+summary: The License Manager Agent snap
+adopt-info: metadata
+license: MIT
+description: |
+  The License Manager Agent Snap deploys the License Manager Agent Python package on your host system. This agent is an essencial component of the License Manager suite.
+
+  This snap requires a few configuration values to be set before it can be used. These values are:
+  - backend-base-url: The URL of the License Manager API server where the agent will send its data. Setting/unsetting this value is more interesting when using the snap in a development environment; do not change it otherwise.
+
+  - oidc-domain: The domain of the OIDC server that the agent will use for authentication. Setting/unsetting this
+    value is more interesting when using the snap in a development environment; do not change it otherwise.
+
+  - oidc-client-id: The client ID of the OIDC application that the agent will use for authentication.
+
+  - oidc-client-secret: The client secret of the OIDC application that the agent will use for authentication.
+
+  - sacctmgr-path: The absolute path to the *sacctmgr* command on the host system. This is optional and defaults to /usr/bin/sacctmgr.
+
+  - scontrol-path: The absolute path to the *scontrol* command on the host system. This is optional and defaults to /usr/bin/scontrol.
+
+  - squeue-path: The absolute path to the *squeue* command on the host system. This is optional and defaults to /usr/bin/squeue.
+
+  - lmutil-path: Absolute path to the binary for lmutil (needed for FlexLM licenses). Defaults to /usr/local/bin/lmutil.
+
+  - rlmutil-path: Absolute path to the binary for rlmutil (needed for RLM licenses). Defaults to /usr/local/bin/rlmutil.
+
+  - lsdyna-path: Absolute path to the binary for lstc_qrun (needed for LS-Dyna licenses). Defaults to /usr/local/bin/lsdyna.
+
+  - lmxendutil-path: Absolute path to the binary for lmxendutil (needed for LM-X licenses). Defaults to /usr/local/bin/lmxendutil.
+
+  - olixtool-path: Absolute path to the binary for olixtool (needed for OLicense licenses). Defaults to /usr/local/bin/olixtool.
+
+  - reservation-identifier: Reservation name used when doing the reconciliation. Defaults to *license-manager-reservation*.
+
+  - lm_user: System user name for creating reservations on behalf of. Defaults to *license-manager*.
```

Comment:
```suggestion
  - lm_user: System user name used when creating reservations. Defaults to *license-manager*.
```

Reply:
modified in [13c89e3](https://github.com/omnivector-solutions/license-manager/pull/389/commits/13c89e37e4fca4d4f8cd3de0090c4f227cc7a738)